base/BUILD.gn

```diff
@@ -1215,7 +1215,8 @@ component("base") {
 
   # Allow more direct string conversions on platforms with native utf8
   # strings
-  if (is_apple || is_chromeos || is_castos || is_cast_android || is_fuchsia) {
+  if (is_apple || is_chromeos || is_castos || is_cast_android || is_fuchsia ||
+      use_cobalt_customizations) {
```

Comment:
I think we only want to say we have native utf8 when we're building for starboard (i.e. use `is_starboard` instead of `use_cobalt_customizations`)